src/runtime/analytics-service.js

```diff
@@ -191,11 +191,18 @@ export class AnalyticsService {
     this.doc_.getBody().removeChild(this.getElement());
   }
 
+  /**
+   * @return {!AnalyticsContext}
+   */
+  getContext() {
+    return this.context_;
+  }
+
   /**
    * @param {!../proto/api_messages.AnalyticsEvent} event
    * @return {!AnalyticsRequest}
    */
-  createLogRequest_(event) {
+  createLogRequest(event) {
```

Comment:
I am unable to load our conversation. But I thought in exporting the getContext method, we could leave this method be private and unchanged. 

Did you mean to leave the _ in line 206 at the end of createLogRequest()?

Reply:
My mistake!